### PR TITLE
Working build

### PR DESCRIPTION
**Pull request by griffinstiens**
- `!stonk [ticker]` will now return `Current Price, Open Price, Previous Closing Price, High, Low` for that ticker
- You may also pass `current, open, close, high, low` after the ticker parameter to retrieve just that data point
- Bot will return an error message in the chat if you pass an invalid ticker. I was trying to check this based on HTTP response code, but even if you pass an invalid ticker to the API, it still responds with HTTP 200 - So I just check for an empty JSON object. Not sure how elegant that implementation is but that was the first thing that came to my mind.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -25,43 +29,44 @@ async def on_member_join(member):
...
-    # if message.content == '!stonks':
-    #     response = stonk_response
-    #     await message.channel.send(response)
-    # elif message.content == 'raise-exception':
-    #     raise discord.DiscordException
-
 # !stonk [ticker] - open, close data
 @bot.command(name='stonk')
-async def grab_stonk_history(ctx, ticker: str):
+async def grab_stonk_history(ctx, ticker: str, price = None):
```
Comment (dcargs) [inline bot.py:34]: price = None;

Default arguments are bae

File: bot.py [hunk 2]
```diff
@@ -25,43 +29,44 @@ async def on_member_join(member):
...
-    if data.empty == True:
-        response = "That ticker doesn't exist --- Please enter a valid ticker symbol"
-        await ctx.channel.send(response)
-        return
-    data_info = data.info()
 
-    open_cost = data.Open
-    close_cost = data.Close
+    query = {'symbol': ticker, 'token': FINN_API_KEY}
+    res = requests.get('https://finnhub.io/api/v1/quote', params=query)
```
Comment (dcargs) [inline bot.py:39]: https://medium.com/@masnun/always-use-a-timeout-for-http-requests-de4da538b9e3#:~:text=And%20in%20consuming%20them%2C%20while,to%20respond%2C%20let's%20move%20on.

Check out that article on why specifying timeouts is important.. I've shot myself in the foot before for not. It should just be an argument in the requests.get function

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,4 @@
 discord
 python-dotenv
 python-decouple
+finnhub-python
```
Comment (dcargs) [inline requirements.txt:4]: Nice find on finnhub. It seems super sweet!

**Comment by griffinstiens** [comment]
YEET

**Pull request merged**